bundles/org.dataflowanalysis.analysis/src/org/dataflowanalysis/analysis/core/FlowGraphCollection.java

```diff
@@ -37,6 +38,11 @@ public FlowGraphCollection(ResourceProvider resourceProvider) {
         this.resourceProvider = resourceProvider;
         this.transposeFlowGraphs = this.findTransposeFlowGraphs();
     }
+    public FlowGraphCollection(ResourceProvider resourceProvider, TransposeFlowGraphFinder transposeFlowGraphFinder) {
+        this.resourceProvider = resourceProvider;
+        this.transposeFlowGraphFinder = transposeFlowGraphFinder;
+        this.transposeFlowGraphs = this.findTransposeFlowGraphs();
+    }
```

Comment:
Use delayed constructor `FlowGraphCollection()` and `initialize(ResourceProvider resourceProvider)`